### PR TITLE
Rental Side all bugs fixed

### PR DESCRIPTION
**Pull request by shlsystemica**
All the bugs of rental sides are fixed including signup & login everything

### CODE REVIEW

File: package-lock.json
Comment (TeamSystemica) [inline package-lock.json:1]: @shlsystemica - i see you have upgraded versions of many packages & dependencies - any reason?

File: src/components/PropertyCreate/PropertyInfo.js [hunk 1]
```diff
@@ -1326,13 +1586,17 @@ function PropertyInfo() {
...
                             }}
-                            value="1 Bike"
+                            value="1"
                           >
                             1 Bike
                           </option>
-                          <option value="2 Bikes">2 Bikes</option>
-                          <option value="Included with Car">Included with Car</option>
-                          <option value="">No Bike Parking</option>
+                          <option value="2">2 Bikes</option>
```
Comment (TeamSystemica) [inline src/components/PropertyCreate/PropertyInfo.js:1593]: why are we just not changing the enum itself?
why are we doing this roundabout mapping? @shlsystemica

File: package-lock.json
Comment (shlsystemica) [reply package-lock.json:1]: I just added one package, everything else is same as previous and that added is package is loader

File: src/components/PropertyCreate/PropertyInfo.js (hunk 1 above)
Comment (shlsystemica) [reply src/components/PropertyCreate/PropertyInfo.js:1593]: It was already there, and all these validation is supplies by api only, maybe i can not define enum itself

**Pull request closed without merging**